File: yggdrasil/examples/split_and_merge/src/split_and_merge_modelD.jl
using Printf

function modelD_function(in_val1, in_val2)
  out_val = in_val1 + in_val2
  @printf("modelD_function(%s, %s) = %s\n", in_val1, in_val2, out_val)
  return in_val1, in_val2, out_val
end

File: yggdrasil/examples/split_and_merge/src/split_and_merge_modelD.m
function [in_val1, in_val2, out_val] = split_and_merge_modelD(in_val1, in_val2)
  out_val = in_val1 + in_val2;
  disp(sprintf('modelD_function(%f, %f) = %f', in_val1, in_val2, out_val));
end
